src/Settings/UnitsSettings.cc

```diff
@@ -23,7 +23,7 @@ DECLARE_SETTINGSFACT_NO_FUNC(UnitsSettings, horizontalDistanceUnits)
         // Distance/Area/Speed units settings can't be loaded from json since it creates an infinite loop of meta data loading.
         QStringList     enumStrings;
         QVariantList    enumValues;
-        enumStrings << "Feet" << "Meters";
+        enumStrings << QObject::tr("Feet") << QObject::tr("Meters");
```

Comment:
In this file, "tr(" is used.
Is there any reason to clarify the "QObject::tr(" class in this process?
I thought it would be better to combine it with other processes.
Also, there are only 5 CPP language files that use this notation.